### PR TITLE
Added Emily and made suggestion for readme

### PR DESCRIPTION
**Pull request by kraus-s**
Emily sent me her bio, so I added it to the "about".

Also, I tried and add a simplified/tl;dr version of how to edit to the readme.

### CODE REVIEW

**Review by ElineElmiger — COMMENTED** [review]
looks good, structure of the readme seems clear

File: README.md [hunk 1]
```diff
@@ -11,7 +11,33 @@ If you're interested in the research network, see our [pages](https://network-di
 more information. Or get in touch with one of the network members.
 
 
-## How to Contribute to the Pages
+## How to Contribute to the Pages (Step by Step)
+
+The basis of the pages are markup-files. If you want to understand how markup files are rendered as a web page, see the technical explanation below.
```
Comment (BalduinLandolt) [inline README.md:16]:
```suggestion
The basis of the pages are markdown files. If you want to understand how markdown files are rendered as a web page, see the technical explanation below.
```

File: README.md [hunk 2]
```diff
@@ -11,7 +11,33 @@ If you're interested in the research network, see our [pages](https://network-di
...
+
+The basis of the pages are markup-files. If you want to understand how markup files are rendered as a web page, see the technical explanation below.
+This section will give you brief explanation on how to edit what is visible to the public in a way that is almost as easy as writing in the text editor of your choice.
+
+### Step One
+
+The most important thing is to first create a branch in GitHub for your edit. This makes sure there is some form of editorial oversight and changes
+can be reviewed before being published. You can find out how to create a branch [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository)
+
+When creating a branch, give it a name like this: "wip/YOUR_BRANCH_NAME".
```
Comment (BalduinLandolt) [inline README.md:24]: thinking about it: maybe we don't even need `wip/`. There won't ever be any non work-in-progress branches here, except main. What do you think?

File: README.md [hunk 3]
```diff
@@ -11,7 +11,33 @@ If you're interested in the research network, see our [pages](https://network-di
...
+
+### Step Two
+
+Edit the file in the branch you just created. You can do this locally or directly in the browser. All files are found in the folder "content".
+Editiing markdown is very easy: You use "#" to mark headings and just wirte your text underneath. If you need more formatting options, check out
+the resources listed in the section on markdown below.
```
Comment (BalduinLandolt) [inline README.md:30]:
```suggestion
Edit the file in the branch you just created. You can do this locally (if you have cloned the repository to your computer) or directly in the browser. All files are found in the folder "content".
Editing markdown is very easy: You use "#" to mark headings and just write your text underneath. If you need more formatting options, check out
the resources listed in the section on markdown below.
```

File: README.md (hunk 1 above)
Comment (BalduinLandolt) [reply README.md:16]: technically, markdown is markup, so it's not wrong... but mixing them may seem confusing, right?

File: content/pages/about.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ workshop "Digital Research in Old Norse Studies: Data Management and Infrastruct
 
 Emily Lethbridge
 
-[Icelandic Saga Map](http://sagamap.hi.is/is/) / Stofnun Árna Magnússonar í íslenskum fræðum, Reykjavík
+Emily works on medieval Icelandic manuscripts and literary culture, landscape and place-names. Digital projects: [Icelandic Saga Map](http://sagamap.hi.is/is/) and [Nafnið.is](https://nafnið.is/)
```
Comment (BalduinLandolt) [inline content/pages/about.md:57]: so far we always had one line name, one line affiliation/infrastructures and then a paragraph bio. I don't mind giving up that schema... but I'm a bit a sucker for consistency. ;-)

**Review by BalduinLandolt — APPROVED** [review]
Over all, looks good, thanks! The readme section is definitely a big improvement!  
I spotted two typos you may want to fix; otherwise some remarks that you may apply or ignore as you see fit.

File: content/pages/about.md (hunk 4 above)
Comment (kraus-s) [reply content/pages/about.md]: Fair. Changed accordingly

**Pull request merged**